src/main/scala/zio/sqs/producer/Producer.scala

```diff
@@ -117,7 +117,7 @@ object Producer {
                       )
                       .map(chunks => reqBuilder(chunks.toList))
                       .mapZIOParUnordered(settings.parallelism)(reqRunner)
-      _          <- stream.runDrain.fork
+      _          <- stream.runDrain.forkDaemon
```

Comment:
How about `forkScoped`?

Reply:
IIUC finalizers run in reverse order(last registered run first).
So fiber will be interrupted before queues and some messages will be lost.

Reply:
Right, and the fiber should terminate anyway when the queues are shutdown. LGTM